src/runtimes/node/utils/included_files.ts

```diff
@@ -53,9 +53,14 @@ export const getPathsOfIncludedFiles = async (
     cwd: basePath,
     dot: true,
     ignore: excludePatterns,
-    followSymbolicLinks: true,
+    followSymbolicLinks: false,
+    // get directories as well to get symlinked directories,
+    // to filter the regular non symlinked directories out mark them with a slash at the end to filter them out.
```

Comment:
Can you move this to L60 so it's above the related property (`markDirectories`)?

Reply:
The related property is `onlyFiles`

Reply:
`get directories as well`

Reply:
- "Get directories" = `onlyFiles`
- "Mark them with a slash" = `markDirectories`

Just trying to make sure these properties are easy to understand when people read the code.